chromium_src/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc

```diff
@@ -52,7 +52,7 @@ String PluginReplacementName(std::mt19937_64* prng) {
 
 void FarblePlugins(DOMPluginArray* owner,
                    HeapVector<Member<DOMPlugin>>* dom_plugins) {
-  LocalFrame* frame = owner->GetFrame();
+  LocalFrame* frame = owner->DomWindow()->GetFrame();
```

Comment:
cc @pilgrim-brave maybe also check `owner->DomWindow() != nullptr`

Reply:
can't hurt

Reply:
Fixed in de34845572138db27937115bb331702e8b407c83